src/Sylius/Behat/Page/External/PaypalExpressCheckoutPage.php

```diff
@@ -97,4 +75,29 @@ protected function verifyUrl(array $urlParameters = [])
             throw new UnexpectedPageException(sprintf('Expected to be on "%s" but found "%s" instead', $this->getUrl($urlParameters), $this->getSession()->getCurrentUrl()));
         }
     }
+
+    /**
+     * @return TokenInterface
+     *
+     * @throws \RuntimeException
+     */
+    private function findCaptureToken()
+    {
+        $tokens = $this->securityTokenRepository->findAll();
+
+        $captureToken = null;
+        foreach ($tokens as $token) {
+            if (strpos($token->getTargetUrl(), 'capture')) {
+                $captureToken = $token;
+
+                break;
+            }
+        }
+
+        if (null === $captureToken) {
```

Comment:
And then we can delete this condition. If it won't return any token it should always throw an exception